mmv1/third_party/terraform/services/compute/resource_compute_instance_template_test.go.tmpl

```diff
@@ -4405,3 +4455,30 @@ resource "google_compute_instance_template" "foobar" {
 `, context)
 }
 {{- end }}
+
+func testAccComputeInstanceTemplate_keyRevocationActionType(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_instance_template" "foobar" {
+  name         = "%{instance_name}"
+  machine_type = "e2-medium"
+
+  disk {
+	source_image = data.google_compute_image.my_image.self_link
+	auto_delete  = true
+	disk_size_gb = 10
+	boot         = true
+  }
+
+  network_interface {
+	network = "default"
+  }
+
+  %{key_revocation_action_type}
```

Comment:
I think this might be why the field is not being detected in these tests.

```suggestion
  key_revocation_action_type = "%s"
```

Reply:
This needs some more changes outside of this suggestion. Will change it tough